src/OpenCensus.Abstractions/Trace/Export/IHandler.cs

```diff
@@ -27,6 +28,7 @@ public interface IHandler
         /// Exports the list of spans to the backend.
         /// </summary>
         /// <param name="spanDataList">Collection of spans to export.</param>
-        void Export(IEnumerable<ISpanData> spanDataList);
+        /// <returns>A <see cref="Task"/> representing the asynchronous export operation.</returns>
+        Task Export(IEnumerable<ISpanData> spanDataList);
```

Comment:
can you please follow naming convention and call method `ExportAsync`.

Reply:
done